internal/main.go

```diff
@@ -45,6 +45,7 @@ func main() {
 	pflag.Lookup("debug").Hidden = true
 	pflag.BoolVarP(&options.Strict, "strict", "s", false, "fail on any warning")
 	pflag.BoolVarP(&options.Pretty, "pretty", "p", false, "output formatted json")
+	pflag.BoolVarP(&options.Raw, "raw", "r", false, "output only Ignition config, without any wrapper")
```

Comment:
Do you think we should mention this is only valid on `openshift` variants or maybe throw a warning/error if we see it set in other variants? I'm not sure that `without any wrapper` means anything without context.

Reply:
Changed it to `never wrap in a MachineConfig; force Ignition output`.  For the other variants, we always output raw Ignition, so it feels odd to complain if the user redundantly requests it.